src/ResourceTemplateValidator.js

```diff
@@ -129,5 +131,40 @@ const validateTemplateRefsExist = async (resourceTemplate) => {
   return []
 }
 
+/**
+ * Validates that all value template refs have unique property URIs.
```

Comment:
Should this be *all  value template refs have unique resource URIs*?

Reply:
Yup. Fixed.